pkg/commands/artifact/option.go

```diff
@@ -56,6 +57,12 @@ func (c *Option) Init() error {
 		return err
 	}
 
+	//--list-all-pkgs option is available only with --format json.
```

Comment:
there is an `artifact/option_test.go`. Could you add tests to that file

Reply:
Done